### DISCUSSION TITLE
Get area/location of listing as a return type?

### DISCUSSION THREAD

**Discussion by puddletowntom** [AnthonyBloomer/daftlistings#161] [General]
The function listings = daft.search() will return a number of things that you can view as a dictionary however I cant seem to return the area or location. On the daft API website, it seems as though there is a return type for area with ID. However the ID from daft.search() just seems to be the ID of the webpage. 

I want to search all properties and was able to do this by not specifying location. This is fast and only requires one request. If I specify the location, then i obviously know the location but it takes too long and requires too many unnecessary requests to get all properties. So the work around was to not specify the location, get everything at once and then identify location by parsing the title string. However this sometimes gets inaccurate results because some addresses are wrong and some addresses overlap with other counties. I tried using the coordinates but it takes too long to compare geo coordinates for every listing. 

If you could just return the location for each listing it would be a lot easier. Does anyone know how to do this? Does return ID mean anything in regards to location?

**Comment by sdfordham** [comment]
Hi I missed this conversation somehow, not sure if it's still of interest to you but anyway I believe it is up to the advertiser to give sufficiently accurate info in the advert about the location and in some kind of sensible format so you can end up with any kind of thing in the address sometimes, e.g. just a street with no number I've seen this before in adverts. So I think the upshot is that the payload returned can be in many different formats.
